tests/python/kaolin/ops/test_mesh.py

```diff
@@ -339,6 +339,20 @@ def test_adjacency_matrix_dense(device, dtype):
                              [1, 1, 0, 0, 0]], dtype=torch.float, device=device)
     assert torch.equal(output, expected)
 
+@pytest.mark.parametrize('device, dtype', FLOAT_TYPES)
+def test_adjacency_consistent(device, dtype):
+    test_mesh = obj.import_mesh('tests/samples/model.obj')
```

Comment:
`obj` isn't been imported.

We recommend avoiding using relative path in case test are launched from a different place, you can use the following approach: https://github.com/NVIDIAGameWorks/kaolin/blob/master/tests/python/kaolin/io/test_obj.py#L22